src/components/BottomNav/BottomNav.js

```diff
@@ -1,22 +1,22 @@
-import React from "react";
```

Comment:
I changed my Prettier to use single quotes and also changed the routing in the bottom nav to route to "/Profile" instead of "/Movies"